backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/ddl/functions/Docs_Sales_TransportOrder_Description.sql

```diff
@@ -0,0 +1,60 @@
+/*
+ * #%L
+ * de.metas.fresh.base
+ * %%
```

Comment:
why we need this?
Where is used?

Reply:
replace the SQL select with a new function here: backend/de.metas.fresh/de.metas.fresh.base/src/main/jasperreports/de/metas/docs/sales/shippingorder/report_description.jrxml